package-lock.json

```diff
@@ -77,21 +77,21 @@
       }
```

Comment:
Please remove these files from this commit/PR. We'll update once a major version change is needed.

Reply:
Should I copy the package.json from the original repo(`commudle/commudle-ng`) and make a commit in mine and again send a PR?